### ISSUE TITLE
 Unable to drag and drop independent node.

### ISSUE THREAD

**Issue by vgouth** [Azure/LogicAppsUX#5265]
### Describe the Bug with repro steps 

1. Create a standard logic app in VS code and open the designer. 
2. Add request trigger. 
3. Add some actions of your choice which are not dependent on other(independent). 
4. Try drag and drop the independent node up and down. 

**Expected:** We should be able to drag and drop the independent node.

**Actual:** **Unable** to drag and drop independent node.

### What type of Logic App Is this happening in? 

Standard (VSCode)

### Are you using new designer or old designer 

New Designer

### Did you refer to the TSG before filing this issue? [https://aka.ms/lauxtsg](<https://aka.ms/lauxtsg>) 

Yes

### Workflow JSON 

```json
{
    "definition": {
        "$schema": "https://schema.management.azure.com/providers/Microsoft.Logic/schemas/2016-06-01/workflowdefinition.json#",
        "actions": {
            "Compose": {
                "type": "Compose",
                "inputs": "ada",
                "runAfter": {}
            },
            "Compose_1": {
                "type": "Compose",
                "inputs": "fadfa",
                "runAfter": {}
            },
            "Compose_2": {
                "type": "Compose",
                "inputs": "hert",
                "runAfter": {}
            },
            "Compose_3": {
                "type": "Compose",
                "inputs": "hs",
                "runAfter": {
                    "Compose_1": [
                        "SUCCEEDED"
                    ],
                    "Compose": [
                        "SUCCEEDED"
                    ],
                    "Compose_2": [
                        "SUCCEEDED"
                    ]
                }
            },
            "List_files_in_folder": {
                "type": "ApiConnection",
                "inputs": {
                    "host": {
                        "connection": {
                            "referenceName": "dropbox"
                        }
                    },
                    "method": "get",
                    "path": "/datasets/default/folders/@{encodeURIComponent(encodeURIComponent('6e202211-2856-4d17-9ded-5beb8b8626b0'))}"
                },
                "runAfter": {
                    "Compose_3": [
                        "SUCCEEDED"
                    ]
                },
                "metadata": {
                    "6e202211-2856-4d17-9ded-5beb8b8626b0": "/"
                }
            },
            "For_each": {
                "type": "foreach",
                "foreach": "@body('List_files_in_folder')",
                "actions": {
                    "Compose_4": {
                        "type": "Compose",
                        "inputs": "@item()?['DisplayName']"
                    }
                },
                "runAfter": {
                    "List_files_in_folder": [
                        "SUCCEEDED"
                    ]
                }
            },
            "Compose_5": {
                "type": "Compose",
                "inputs": "jds",
                "runAfter": {
                    "For_each": [
                        "SUCCEEDED"
                    ]
                }
            }
        },
        "contentVersion": "1.0.0.0",
        "outputs": {},
        "triggers": {
            "When_a_HTTP_request_is_received": {
                "type": "Request",
                "kind": "Http"
            }
        }
    },
    "kind": "Stateful"
}
```

### Screenshots or Videos

[https://github.com/user-attachments/assets/ea2774e9-fcbe-4b7f-8f89-2d82ba7249ba](<https://github.com/user-attachments/assets/ea2774e9-fcbe-4b7f-8f89-2d82ba7249ba>)

![drag and drop1](<https://github.com/user-attachments/assets/cbf84ef6-a7a0-4e60-9277-ac9e372c56f3>)

### Browser

VS Code

### Additional context

VSIX Version: 4.44.5

**Comment by ccastrotrejo** [comment]
This issue is related to an update in the vscode app, the workaround for this and the new implementation that vscode came up with is Pressing Shift while dragging and dropping, this will re-enable the behavior.

**Comment by niphlod** [comment]
confirmed, shift pressing restores the behaviour ... jm2c: I'd vote to revert this if possible

**Comment by ccastrotrejo** [comment]
Hi @niphlod, this issue is coming from the vscode app. Here is the main issue. https://github.com/Azure/LogicAppsUX/issues/5265, , we will keep track of it to be aware of their changes. 
So far the issue has been identified in the 1.92.0 version. Version 1.91.1 seems to work fine. Another workaround is to install the 1.91.1 version

**Comment by ccastrotrejo** [comment]
This is now fixed in vscode 1.92.1.
